content/updates/2021-04/1-Review-your-proposals-more-efficiently.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_img: ./img/revamped-proposal-screen.png
+pim_announcement_alt_img: "New proposals screen"
+pim_announcement_audience:
+- EE
+---
+
+# Review your proposals more efficiently
+::: meta-data type="Improvement" features="Productivity, Collaboration" available="late April" in="EE" link-to-doc="../articles/review-products-proposals.html"
+
+We're glad to introduce the revamp of our proposals screen. It's now even more practical and efficient for those of you who, depending on their catalog rights, need to review the enrichment work of teammates. 
```

Comment:
We are pleased to present to you our new screen to manage proposals. This new screen has been thought to be more simple to use and straightforward and to make it easy and faster for those of you who, depending on your catalog rights, need to review your teammates or external contributors suggestions.

Reply:
I would better suggest this, as the screen is only useful for those who have the own rights on some categories.
```suggestion
We are pleased to present to you our new screen to manage proposals. This screen will make it easier and faster for those of you who, depending on your catalog rights, need to review your teammate or external contributor suggestions. Indeed, it has been thought to be more simple and straightforward to use. 
```

Reply:
```suggestion
We are happy to introduce our new screen to manage proposals. This screen will make it easier and faster for those of you who, depending on your catalog rights, need to review your teammate or external contributor suggestions. Indeed, it has been designed to be more simple and straightforward to use.
```